digiwf-alw-integration/src/main/java/io/muenchendigital/digiwf/alw/integration/infrastructure/streaming/MessageProcessor.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c): it@M - Dienstleister für Informations- und Telekommunikationstechnik
+ * der Landeshauptstadt München, 2020
+ */
+package io.muenchendigital.digiwf.alw.integration.infrastructure.streaming;
```

Comment:
I usually place the configs for spring cloud stream consumers and producers in the api package. They are kinda similar endpoints to rest controllers.

Reply:
OK, youre right. Done